### PR TITLE
Revise sin

### PR DESCRIPTION
**Pull request by bszmelcz**
### Details:
 - rearrange spec to meet criteria
 - add Sin to list of unary operators in unit test file

### Tickets:
 - 37478

### CODE REVIEW

File: docs/ops/arithmetic/Sin_1.md [hunk 1]
```diff
@@ -6,6 +6,18 @@
...
 **Short description**: *Sin* performs element-wise sine operation with given tensor.
 
+**Detailed description**: *sin* does the following with the input tensor *a*:
+\f[
+a_{i} = sin(a_{i})
+\f]
+
+The sine of an acute angle theta is defined in the context of a right triangle: for the specified angle, it is the ratio of the length of the side 
+that is opposite that angle (the opposite), to the length of the longest side of the triangle (the hypotenuse).
+
```
Comment (jdanieck) [inline docs/ops/arithmetic/Sin_1.md:16]: Did you invent this description on your own or copy-pasted ? :)

File: docs/ops/arithmetic/Sin_1.md [hunk 2]
```diff
@@ -22,11 +34,6 @@
```
Comment (jdanieck) [inline docs/ops/arithmetic/Sin_1.md:34]: Please add information about supported input tensor ranks. Are they of arbitrary size or there are some limitations?

File: docs/ops/arithmetic/Sin_1.md (hunk 1 above)
Comment (bszmelcz) [reply docs/ops/arithmetic/Sin_1.md:16]: both :), but it is the easiest definition of the sine that I can think of.

Comment (bszmelcz) [reply docs/ops/arithmetic/Sin_1.md:16]: It would make much more sense if there was an option to add a image of the triangle to the documentation

Comment (jdanieck) [reply docs/ops/arithmetic/Sin_1.md:16]: So maybe just redirect to [Wikipedia](https://en.wikipedia.org/wiki/Sine) ?

Comment (bszmelcz) [reply docs/ops/arithmetic/Sin_1.md:16]: if that is ok then sure

**Review by jdanieck — APPROVED** [review]

File: docs/ops/arithmetic/Sin_1.md [hunk 3]
```diff
@@ -6,13 +6,20 @@
 
 **Short description**: *Sin* performs element-wise sine operation with given tensor.
 
+**Detailed description**: *sin* does the following with the input tensor *a*:
+\f[
+a_{i} = sin(a_{i})
+\f]
+
+The best description of the sine operation can be found on [Wikipedia](https://en.wikipedia.org/wiki/Sine)
```
Comment (lazarevevgeny) [inline docs/ops/arithmetic/Sin_1.md:14]: Let's remove this line. The sine operation is obvious. But we should specify what is the input to the function: radians or degrees?

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**